### PR TITLE
Joshdellaz/feature/adjustable baud rate delay

### PR DESCRIPTION
**Pull request by jlazzz**

### CODE REVIEW

File: lib/driver/sdr_driver.c [hunk 1]
```diff
@@ -88,3 +91,13 @@ os_task_return_t sdr_rx_task(void *param) {
...
     }
 }
+
+int sdr_uhf_set_rf_mode(uint8_t rf_mode){
+    csp_iface_t * iface = csp_iflist_get_by_name(SDR_IF_UHF_NAME);
+    if(iface == NULL){
+        return -1;
+    }
+    sdr_interface_data_t * sdr_ifdata = iface->interface_data;
+    sdr_ifdata->sdr_conf->uhf_conf.uhf_baudrate = get_uhf_baud_t_from_rf_mode_number(rf_mode);
```
Comment (Taylorr82) [inline lib/driver/sdr_driver.c:94]: That's a scary pointer

File: lib/driver/sdr_driver.c [hunk 2]
```diff
@@ -88,3 +91,13 @@ os_task_return_t sdr_rx_task(void *param) {
         }
     }
 }
+
+int sdr_uhf_set_rf_mode(uint8_t rf_mode){
+    csp_iface_t * iface = csp_iflist_get_by_name(SDR_IF_UHF_NAME);
```
Comment (Taylorr82) [inline lib/driver/sdr_driver.c]: Has this been tested on ground station? It's pretty picky about things

File: lib/driver/sdr_driver.c (hunk 1 above)
Comment (jlazzz) [reply lib/driver/sdr_driver.c:94]: How so?

File: lib/driver/sdr_driver.c (hunk 2 above)
Comment (jlazzz) [reply lib/driver/sdr_driver.c]: Just tested it; works fine

File: lib/driver/sdr_driver.c (hunk 1 above)
Comment (Taylorr82) [reply lib/driver/sdr_driver.c:94]: It's long lol

Comment (jlazzz) [reply lib/driver/sdr_driver.c:94]: Yeah. Tell that to Ron

**Comment by Taylorr82** [comment]
This needs to be rebased on master to compile on ground station. Other than that, it seems fine

File: lib/driver/sdr_driver.c [hunk 3]
```diff
@@ -3,6 +3,9 @@
 #include "fec.h"
 #include "osal.h"
 #include <stdio.h>
+#include "rfModeWrapper.h"
+#include "csp_types.h"
```
Comment (Taylorr82) [inline lib/driver/sdr_driver.c]: I don't like that we included CSP in this repo.. I wonder if there's a better place to put it

**Review by Taylorr82 — APPROVED** [review]

**Pull request merged**